pydantic/errors.py

```diff
@@ -2,22 +2,64 @@
 
 import re
 
+from typing_extensions import Literal
+
+from .version import VERSION
+
 __all__ = (
     'PydanticUserError',
-    'PydanticSchemaGenerationError',
     'PydanticUndefinedAnnotation',
+    'PydanticSchemaGenerationError',
     'PydanticInvalidForJsonSchema',
 )
 
+# TODO set up a cloudflare worker to redirect to the correct page
+DEV_ERROR_DOCS_URL = f'https://e.pydantic.dev/{VERSION}/d/'
```

Comment:
I'm assuming in `e.pydantic.dev` the `e` is for errors? Is this better than `errors.pydantic.dev` or `pydantic.dev/errors` or similar? Also what's the `/d/` for? 

If you feel strongly about the `e.pydantic.dev` being right, it's not very important to me, just curious.

Reply:
The `d` for for developer errors - to differentiate from validation errors which I would guess should get the prefix `/v/`.

The `e.pydantic.dev` was to keep the url in the error message short, but we could use `errors`.

More generally, the rationale of using a custom subdomain and building a service to deal with these was three fold:
1. (most importantly) to give us flexibility about what we do with this error documentation in the future - once a version of pydantic is deployed with this feature, we can't update the URL so we want a way to direct people wherever we want without it constraining how we host the rest of the docs.
2. to keep the URL short and clear - I guess `errors.` improves clarity significantly with minimal increase in length.
3. to allow us to log analytics on this in future if we want to